src/main/java/org/springframework/data/tarantool/core/TarantoolTemplate.java

```diff
@@ -88,6 +99,67 @@ public TarantoolTemplate(
         );
     }
 
+    private <T> CallResultMapper<T, SingleValueCallResult<T>>
+    getDefaultWithTupleResultMapper(Class<T> entityClass, Optional<TarantoolSpaceMetadata> spaceMetadata) {
+        // blocked because it's not deepcopy
+        // MessagePackMapper defaultConverterWithTupleConverter = tarantoolClient.getConfig().getMessagePackMapper().copy();
+        MessagePackMapper defaultConverterWithTupleConverter = DefaultMessagePackMapperFactory.getInstance().defaultComplexTypesMapper();
```

Comment:
It seems that this is also worth starting a ticket or discussing

Reply:
It is done in another way -- you create a new mapper and populate it with one from the config. There should be examples in tests or some other code around.

Reply:
So this is the question of how to populate it, there is a copy constructor
https://github.com/tarantool/cartridge-java/blob/a9bfb28812448e1298e2e23554f2f39b27ecddb7/src/main/java/io/tarantool/driver/mappers/DefaultMessagePackMapper.java#L43-L53
but it doesn't do a deep copy. We could run through the converters, but this is a bad option and we don’t have access to the converters. Perhaps I did not understand you, please write in more detail what you meant or give an example

Reply:
Discussed with @vrogach2020 that we should leave it as it is, not reuse the custom one until we fix this problem in [cartridge-java#166](https://github.com/tarantool/cartridge-java/issues/166).

Reply:
Described problem in #84